### PR TITLE
use actions

### PR DESCRIPTION
**Pull request by ijlyttle**
A few changes here:

- uses [renv](https://rstudio.github.io/renv/index.html)
- uses actions builds and deploys to `gh-pages` branch with every commit to `main` branch
- removes "build" files from repo for `main` branch

A couple comments below where I had to modify some of "your" (@3mmaRand) stuff. Nothing too major (I hope)

### CODE REVIEW

File: materials/day-1-session-1-introduction.qmd [hunk 1]
```diff
@@ -773,7 +773,7 @@ devtools::load_all()
 ```
 
 ```{r}
-Loading ussie
+# Loading ussie
```
Comment (ijlyttle) [inline materials/day-1-session-1-introduction.qmd:776]: I had to comment this to get things to build

File: schedule.qmd [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+| Time          | Activity                                                                                 |
+|:----------------------------------|:------------------------------------|
+| 09:00 - 10:30 | [Session 1: Introduction and the whole game](materials/day-1-session-1-introduction.qmd) |
+| 10:30 - 11:00 | *Coffee break*                                                                           |
+| 11:00 - 12:30 | [Session 2: Documentation - Minimal](materials/day-1-session-2-doc-minimal.qmd)          |
+| 12:30 - 13:30 | *Lunch break*                                                                            |
+| 13:30 - 15:00 | [Session 3: Unit testing](materials/day-1-session-3-testing.qmd)                         |
+| 15:00 - 15:30 | *Coffee break*                                                                           |
+| 15:30 - 17:00 | [Session 4: Documentation - Sharing]()                                                   |
```
Comment (ijlyttle) [inline schedule.qmd:21]: `quarto build` scans for `.qmd` links, then renders those automatically (yay!), but if the `.qmd` file is not there, the build fails - so I removed this link for now.

**Comment by 3mmaRand** [comment]
Wow, thanks you so much for sorting all that out Ian! You're a star! Really, really appreciated.

**Review by 3mmaRand — APPROVED** [review]

**Comment by ijlyttle** [comment]
Thanks! Glad it's working!

**Pull request merged**